### DISCUSSION TITLE
Initial Data Load to pocketbase with .sqlite file

### DISCUSSION THREAD

**Discussion by kuatroka** [maxpert/marmot#84] [General]
Hi @maxpert  in this demo [video](https://www.youtube.com/watch?v=Zapupe_FREc&t=203s) You say "I loaded up some data to pocketbase" and you share the link to a website with .sqlite files. _**How did you load it**_? As I understand it's only possible to import .json format files. I'm struggling to find an easy way to load an existing .sqlite file to pocketbase. Thanks.

**Comment by maxpert** [comment]
Thats a very good question, probably something that pocketbase community should make even more simpler. Here is what I did:
  - Start pocketbase & configure login etc so that DB is initialized.
  - Created a table using pocketbase with same/similar schema as your source table.
  - Stopped pocketbase.
  - SQLite CLI into pocketbase DB. 
  - Attach the source DB with data rows.
  - Do: `INSERT INTO pocketbase_database (col1, col2, col3...) VALUES SELECT col1, col2, col3, ... FROM attached_db.original_table`

**Reply by kuatroka** [reply]
Agree that it could be much simpler than that. Thanks for the mini guide. I haven't delved into the pocketbase in anger yet, but in your bullet point 4 I guess, the DB credentials are easy to find? (not the pocketbase, but the actual SQLite DB path, etc) Thanks

**Reply by maxpert** [reply]
That's the amazing thing about it since you are coping over the rows into the table, PocketBase will just respect those row values. No creds needed.

**Reply by kuatroka** [reply]
Thanks @maxpert . Sorry, I didn't ask the question correctly. I meant, where do I get the address/path to the SQLite file of the Pocketbase?

After I stop the Pocketbase instance, how do I "CLI SQLite into Pocketbase" ?
Usually I need to know the file path of the SQLite database, then I connect to it. So where do I get the file path of the Pocketbase's SQLite so I can CLI into it?
Thanks

**Reply by maxpert** [reply]
Yes PocketBase has a SQLite file just like anything using SQLite, you can CLI into that.
